rc/.eslintrc

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["standard", "standard-react"]
+  "extends": ["standard", "standard-react"],
+  "rules": {
+    "standard/object-curly-even-spacing": [2, "either"],
```

Comment:
**Note:** npm3 will stop automatically installing `peerDependencies`. [Read more.](https://github.com/npm/npm/blob/master/CHANGELOG.md#peerdependencies) and only show a warning when the peerDependency is not installed by the parent user.